applications/plugins/MultiThreading/src/TaskSchedulerBoost.h

```diff
@@ -165,7 +161,7 @@ namespace sofa
 
 		
 
-			mutable boost::detail::spinlock		mTaskMutex;
+            mutable boost::detail::spinlock		mTaskMutex {BOOST_DETAIL_SPINLOCK_INIT};
```

Comment:
GG for the fix!
I think you should choose between keeping a TAB indentation or fixing it with spaces for the whole file ;-)

Reply:
Well...I'm not in favor of changing the whole file indent for a one line modification. In that case isn't it better to apply the astyle just before merging the PR ? 

Shouldn't I wait for the automatic application of the astyle formatting sheet by the CI ?